rt_bi_emulator:
  em_tg1:
    ros__parameters:
      render: False
      id: 1
      updateInterval: 0.25
      timesSecs:
        - 0.0
        - 50.0
      cutOffTime: inf
      saPoses:
        - "[10.0, 30.0, 0.0]"
        - "[490.0, 70.0, 0.0]"
      centersOfRotation:
        - "[10.0, 30.0]"
        - "[490.0, 70.0]"
      fovs:
        - "[[15.0, 25.0], [25.0, 25.0], [25.0, 35.0], [15.0, 35.0]]"
        - "[[485.0, 65.0], [495.0, 65.0], [495.0, 85.0], [485.0, 85.0]]"
